resources/rdiff_backup.rb

```diff
@@ -124,6 +124,9 @@
 
   cron new_resource.name do
     user node['rdiff-backup']['server']['user']
+    command ['/usr/bin/flock',
+             node['rdiff-backup']['server']['lock_dir'] + '/' + new_resource.name,
+             filename].join(' ')
```

Comment:
IMHO this is more readable.
```suggestion
    command [
        '/usr/bin/flock',
        node['rdiff-backup']['server']['lock_dir'] + '/' + new_resource.name,
        filename
    ].join(' ')
```